src/main/java/seedu/algobase/model/AlgoBase.java

```diff
@@ -176,6 +176,7 @@ public void removeTag(Tag key) {
 
     /**
      * Replaces the contents of the Plan list with {@code plans}.
+     * {@code plans} must not contain duplicate plans.
```

Comment:
I think we don't check for duplicate plans?

Reply:
We don't allow plans with duplicate names for now.